Adyen/Utilities/Debugging/Assertion.swift

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// A typealias for a closure that handles a URL through which the application was opened.
+/// :nodoc:
+public typealias AssertionListner = (String) -> Void
+
+// :nodoc:
+public enum AdyenAssertion {
+
+    public static var listner: AssertionListner?
```

Comment:
maybe this should be `internal` if its only accessed by unit tests.

Reply:
Good point 🤔